cmd/ttn-lw-stack/commands/dr.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	drCommand = &cobra.Command{
+		Use:   "dr",
```

Comment:
Is `dr-db` more consistent, with `is-db` and `ns-db`?

Reply:
Yes, it is more consistent with the rest of the commands, but there is no database here.

Reply:
From a user's perspective, there is a database.